zeroeq/http/server.cpp

```diff
@@ -139,8 +139,9 @@ class Server::Impl : public detail::Sender
         int idSize = 0;
         while( !request.complete( ))
         {
+            int flags = 0;
             // id of client (used for reply)
-            idSize = ::zmq_recv( socket, id, sizeof( id ), 0 );
+            idSize = ::zmq_recv( socket, id, sizeof( id ), flags );
```

Comment:
You're right, this assignment has no effect, yet I was sure it had solved the last of my problems when I did it. I'll remove it and test everything again on Friday.